src/main/java/com/aws/greengrass/secretmanager/SecretManagerIPCAgent.java

```diff
@@ -73,6 +73,7 @@ public void setCloudCallThreadPool(Topics config) {
                 .findOrDefault(DEFAULT_CLOUD_REQUEST_SIZE, CLOUD_REQUEST_QUEUE_SIZE_TOPIC));
         if (cloudCallQueueSize <= 0) {
             logger.atWarn().kv("queueSize", DEFAULT_CLOUD_REQUEST_SIZE)
+                    .kv("configured", cloudCallQueueSize)
```

Comment:
is this a development change or actually something we want to release?

Reply:
Yes, we can keep it. The previous log message doesn't show the configured value.